pkg/apis/servicecatalog/v1beta1/broker.go

```diff
@@ -46,6 +46,16 @@ func (b *ServiceBroker) GetURL() string {
 	return b.Spec.URL
 }
 
+// GetSpec returns the broker spec.
+func (b *ClusterServiceBroker) GetSpec() CommonServiceBrokerSpec {
```

Comment:
Maybe in a follow-up it would be nice to remove GetURL and use this method instead. I like it better, and requires much less boilerplate. 👍